CMakeLists.txt

```diff
@@ -12,7 +12,7 @@
 # publicly and display publicly, and to permit others to do so.
 #------------------------------------------------------------------------------#
 
-cmake_minimum_required(VERSION 3.14)
+cmake_minimum_required(VERSION 3.20)
```

Comment:
Do we need to up the cmake version number? Care should go into this, as I've noticed that some LANL machines are stuck on cmake < 3.20